projboard/templates/projboard/searchArticle/search_article.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/search_article.css' %}">
+    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />    
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Anton&family=Zilla+Slab:wght@400;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+<main>
+    <section>
+
+            <form method="POST" action="">
+                {% csrf_token %}
+                
+                <div class="search_button">
+                    <Label for="titleInput"><h4><strong>Enter Article Title:</strong></h4></Label>
+                    <div class="wrap">
+                        <span style="font-variation-settings: 'FILL' 1, 'wght' 700, 'GRAD' 0, 'opsz' 48;" class="material-symbols-outlined">search</span>
+                        <input id="titleInput" type="text" name="title" placeholder="search article.." value="{{userinput}}">
+                        
+                    </div>
+                    <button id="searchButton" type="submit">Search</button>
+                </div>
+            </form>
+
+    </section>
+
+    <section id="searchsection">
+            <h2>Search Result:</h2>
+            {% if numArticles > 0 %}
+                <div id="searchedArticles">
+                    {% for article in articles %}
+                        <hr>
+                        <article>
+                            <header><p class="nickname"> &#128100; {{article.user_id.nickname}} | &#8987; {{article.date}} </p></header>
+                            <h2>{{article.title | truncatechars:30}}</h2>
+                            <p>{{article.content | truncatechars:70}}</p>
+                            <footer>
```

Comment:
Can we use footer element inside main element?

Reply:
yes I think , it's a convient , just to understand the section parts.